src/sentry/conf/server.py

```diff
@@ -855,6 +855,13 @@ def create_partitioned_queues(name):
     "organizations:integrations-issue-sync": True,
     # Enable interface functionality to receive event hooks.
     "organizations:integrations-event-hooks": False,
+    # Enable integration functionality to work with alert rules
+    "organizations:integrations-alert-rule": False,
```

Comment:
@MeredithAnya Shouldn't the default value be `True` here so on-prem folks get these features by default? I realize `organizations:integrations-event-hooks` probably should be updated to `True` as well.